src/util/transactions.ts

```diff
@@ -10,7 +12,7 @@ export const parseBlockchainTransaction = (tx, address: string) => {
       value: toBitcoin(tx.value.toString()),
       from: tx.senders,
       recipient: tx.recipients,
-      timestamp: tx.time,
+      timestamp: tx.time || getUnixTimestamp(),
```

Comment:
may be better to just show null here? As not not confuse the user.

Reply:
Well the only case when this makes a new timestamp is if is is a brand new transaction which just happened. You think having N/A as it was there before is better than the date now?